boards/arduino/portenta_c33/arduino_portenta_c33.dts

```diff
@@ -0,0 +1,263 @@
+		regulator-boot-on;
+	};
+
+	aliases {
+		led0 = &led1;
+	};
+};
+
+&sci9 {
+	pinctrl-0 = <&sci9_default>;
+	pinctrl-names = "default";
+	status = "okay";
+	uart9: uart {
+		current-speed = <115200>;
+		status = "okay";
+	};
+};
+
+&sci8 {
+	pinctrl-0 = <&sci8_default>;
+	pinctrl-names = "default";
+	status = "okay";
+	uart8: uart {
+		current-speed = <921600>;
+		status = "okay";
+		hw-flow-control;
+
+		bt_hci_uart: bt_hci_uart {
+			compatible = "zephyr,bt-hci-uart";
+			status = "okay";
+
+			esp32 {
+				compatible = "renesas,bt-hci-da1453x";
+				status = "okay";
+				reset-gpios = <&ioport8 4 GPIO_ACTIVE_LOW>;
+				reset-assert-duration-ms = <100>;
+				boot-duration-ms = <2000>;
+			};
+		};
+	};
+};
+
+&option_setting_s {
+	status = "disabled";
+};
+
+&option_setting_sas {
+	status = "disabled";
+};
+
+&option_setting_ofs {
+	status = "disabled";
+};
+
+&iic1 {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	interrupts = <91 1>, <92 1>, <93 1>, <94 1>;
+	interrupt-names = "rxi", "txi", "tei", "eri";
+	clock-frequency = <DT_FREQ_M(1)>;
+	pinctrl-0 = <&iic1_default>;
+	pinctrl-names = "default";
+};
+
+&spi1 {
+	pinctrl-0 = <&spi1_default>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&ioport0 {
+	status = "okay";
+};
+
+&ioport1 {
+	status = "okay";
+};
+
+&ioport2 {
+	status = "okay";
+};
+
+&ioport3 {
+	status = "okay";
+};
+
+&ioport4 {
+	status = "okay";
+};
+
+&ioport5 {
+	status = "okay";
+};
+
+&ioport6 {
+	status = "okay";
+};
+
+&ioport7 {
+	status = "okay";
+};
+
+&ioport8 {
+	status = "okay";
+};
+
+&ioport9 {
+	status = "okay";
+};
+
+&ioporta {
+	status = "okay";
+};
+
+&ioportb {
+	status = "okay";
+};
+
+&xtal {
+	clock-frequency = <DT_FREQ_M(24)>;
+	mosel = <0>;
+	#clock-cells = <0>;
+	status = "okay";
+};
+
+&subclk {
+	status = "okay";
+};
+
+&pll {
+	clocks = <&xtal>;
+	div = <3>;
+	mul = <25 0>;
+	status = "okay";
+};
+
+&usbhs {
+	pinctrl-0 = <&usbhs_default>;
+	pinctrl-names = "default";
+	maximum-speed = "high-speed";
+	status = "okay";
+	zephyr_udc0: udc {
+		status = "okay";
+	};
+};
+
+&usbhs_phy {
+	phys-clock-src = "xtal";
+};
+
+&adc0 {
+	status = "okay";
+	pinctrl-0 = <&adc0_default>;
+	pinctrl-names = "default";
+};
+
+&port_irq9 {
+	interrupts = <41 12>;
+	status = "okay";
+};
+
+&port_irq10 {
+	interrupts = <42 12>;
+	status = "okay";
+};
+
+&pwm1 {
+	pinctrl-0 = <&pwm1_default>;
+	pinctrl-names = "default";
+	interrupts = <63 1>, <64 1>;
+	interrupt-names = "gtioca", "overflow";
+	status = "okay";
+};
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 DT_SIZE_K(64)>;
+			read-only;
+		};
+
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x00010000 (DT_SIZE_M(2) - DT_SIZE_K(192))>;
+		};
+		scratch_partition: partition@1D0000 {
+			label = "image-scratch";
+			reg = <0x0001D0000 DT_SIZE_K(128)>;
+		};
```

Comment:
don't follow this, you have slot 0 and scratch? Scratch shouldn't be used unless required e.g. because slot 0 and slot 1 have different sector sizes... and apart from that there is no slot 1 defined?

Reply:
Fixed, thx